azure-mgmt-dns/src/main/java/com/azure/management/dns/CloudError.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.management.dns;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The CloudError model.
+ */
+@Fluent
+public final class CloudError {
```

Comment:
still CloudError?

Reply:
@xccc-msft You can just delete it. Generator will no longer generate this class.

Reply:
Deleted.